src/my_project/database/gui_repo_worker.py

```
@@ -744,3 +739,37 @@ def save_rating(self, comic_id: str, rating: int) -> None:
+            path: The filepath of the comic archive to be searched against.
+
+        Raises:
+            LookupError: if the comic is not found in the database.
+        """
+        path = Path(path)
+        self.cursor.execute("SELECT id FROM comics WHERE path = ?", (str(path),))
+        result = self.cursor.fetchone()
+        self.conn.close()
+        if result:
+            return result[0]
+        else:
```

Ruff (0.16.1)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)